CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Guidelines for Contribution
+
+## How can I contribute to vscode-seneca-college project?
+
+Never contributed to open source before? Are you curious about how contributions function in our project?
+Here is a brief summary!
+
+**Here is a brief overview of how to contribute to vscode-seneca-college:**
+
+---
+
+1. Start by identifying an issue or a feature that you would want to add. Choose a issue from the issues tab that you would like to work on. Feel free to open a new issue in the project if it is not in the list of issues.
+
+2. In your local GitHub organization, Fork current repository to produce a personal copy of this project by clicking the "Fork" button in the upper right corner of your screen. If you leave the default settings, the repository `underyour-GitHub-username/repository-name` should appear under your profile.
+
+3. Make a local copy of the repository using:
+
+```
+git clone https://github.com/github-username/vscode-seneca-college.
+```
+
+4. Create working branch:
```

Comment:
Don't wanna be supper picky, but you should probably add `a` between the working branch so -> `4. Create a working branch:`